example/Cpp/src/AlarmClock/Clock.lf

```diff
@@ -0,0 +1,206 @@
+/*
+* This is a minimal example of an alarmclock implemeted using the 
+* features lingua franca supplies.
+* 
+* This is just an extract and simplification from the main project
+* which you can find here: https://github.com/revol-xut/lf-alarm-clock
+*
+* Author: Tassilo Tanneberer <tassilo.tanneberger@tu-dresden.de>
+*/
+
+
+target Cpp{
+    cmake-include: "AlarmClock.cmake",
+    keepalive: true
+};
+
+public preamble {=
+    #include "../../../src/shared_header.hpp"
```

Comment:
The prefix is fishy. Simply use `#include shared_header.hpp` in combination with my suggested change in AlarmClock.cmake

Reply:
fixed